packages/web-components/src/option/README.md

```diff
@@ -0,0 +1,139 @@
+# ListboxOption
+
+> An option component is a user interface element that represents a selectable item in a listbox. It provides users with a visual and interactive way to choose from a set of available options. The option component typically includes text or other content that describes the option, and may also include additional metadata or visual cues, such as icons or checkboxes, to help users make informed choices. When selected, the option component triggers an action or updates the state of the listbox or other related components.
+
+<br />
+
+## **Engineering Spec**
+
+An implementation of an option. To avoid namespace collisions with the `Option()` constructor, the component class is `ListboxOption`, and the implementation is named `fluent-option`.
+
+Fluent WC3 `<fluent-option>` component will only provide internals related to form association when used within a form-associated component such as `fluent-select`.
```

Comment:
To my understanding this isn't used in Fluent Select (@procload), it's used in "Dropdown" and "Combobox" though...as well as a "listbox".

Reply:
Also, when would this be used outside of one of these components?

Reply:
@chrisdholt `fluent-select` was a mistype on my part. I've updated it to `fluent-dropbox`.

My thinking with this line was more of a just an FYI for consumers. I just didn't want to make any assumptions about what a dev might do with it. I can remove this line though if you think it's unnecessary or confusing or something.

Reply:
I'd just be sure we're clear and correct on the usage and use case.